e2e/constants/selectors/analytics.ts

```diff
@@ -1,14 +1,20 @@
 export const ANALYTICS_SELECTORS = {
-    NAV_ELEMENTS: {
-        MORE_DROPDOWN: 'more-dropdown-icon',
-        ANALYTICS_TAB: 'analytics-more-tab'
-    },
-    
-    METRIC_ELEMENTS: {
-        VISITS: 'visits-metric',
-        STARTS: 'starts-metric',
-        SUBMISSIONS: 'submissions-metric',
-        COMPLETION_RATE: 'completion-rate-metric',
-        INSIGHTS_COUNT: 'insights-count'
-    }
-};
\ No newline at end of file
+    // Main components
+    insightsPanel: "insights",
+    
+    // Metrics
+    visitsMetric: "visits-metric",
+    startsMetric: "starts-metric",
+    submissionsMetric: "submissions-metric",
+    
+    // Counts
+    insightsCount: "insights-count",
+    
+    // Charts and visualizations
+    visitsChart: "visits-chart",
+    submissionsChart: "submissions-chart",
+    
+    // Time period controls
+    timeRangeSelector: "time-range-selector",
+    dateRangePicker: "date-range-picker"
```

Comment:
Instead of adding comments to separate the selectors on different sections of a page, just create multiple constants.

```suggestion
export const ANALYTICS_SELECTORS = { insightsPanel: "insights" }

export const METRICS_SELECTORS = {
  visitsMetric: "visits-metric",
  startsMetric: "starts-metric",
  submissionsMetric: "submissions-metric"
}

// etc.
```